File: org.summer.view.widget/src/org/summer/view/widget/input/CaptureMode.java
package org.summer.view.widget.input;

public enum CaptureMode {
	Element, // Mouse capture is applied to a single element. Mouse input goes
				// to the captured element.
	None, // No mouse capture. Mouse input goes to the element under the mouse.
	SubTree, // Mouse capture is applied to a subtree of elements. If the mouse
				// is over a child of the element with capture, mouse input is
				// sent to the child element. Otherwise, mouse input is sent to
				// the element with mouse capture.
}

File: summer/WebContent/js/system/windows/input/CaptureMode.js
/**
 * CaptureMode
 */

define(["dojo/_base/declare", "system/Type"], function(declare, Type){
	var CaptureMode = declare("CaptureMode", null,{

	});
	
	CaptureMode.None = 0;
	CaptureMode.Element = 1;
	CaptureMode.SubTree = 2;
	
//	CaptureMode.Type = new Type("CaptureMode", CaptureMode, [Object.Type]);
	return CaptureMode;
});
